diff --git a/include/tcpdf_static.php b/include/tcpdf_static.php
index a118d05..c4657fd 100644
--- a/include/tcpdf_static.php
+++ b/include/tcpdf_static.php
@@ -1906,6 +1906,9 @@ class TCPDF_STATIC {
 		if (preg_match('|^https?://|', $filename) == 1) {
 			return self::url_exists($filename);
 		}
+    if(stripos($filename,'data:')===0) {
+      return true;
+    }
 		if (strpos($filename, '://')) {
 			return false; // only support http and https wrappers for security reasons
 		}
